fastapi_events/handlers/gcp.py

```diff
@@ -0,0 +1,49 @@
+import json
+from typing import Callable, Iterable
+
+from google.cloud import pubsub_v1
+
+from fastapi_events.errors import ConfigurationError
+from fastapi_events.handlers.base import BaseEventHandler
+from fastapi_events.typing import Event
+
+
+def _json_serializer(event: Event) -> str:
+    return json.dumps(event, default=str)
+
+
+class GoogleCloudPubSubHandler(BaseEventHandler):
```

Comment:
@Mohsin-Ul-Islam I believe there will be needs to forward events to multiple topics depending on the event name, as some folks using AWS SNS (the pub/sub offering by AWS) have indicated.

What about renaming `GoogleCloudPubSubHandler` to `GoogleCloudSimplePubSubHandler` to allow room in the future to implement a GCP pub/sub handler with more advanced routing capabilities?

 I think it'd be better if we rename it to something such as `GoogleCloudSimplePubSubHandler` to indicate that the proces